src/vm/docs/mod.rs

```diff
@@ -173,6 +173,66 @@ const TO_INT_API: SimpleFunctionAPI = SimpleFunctionAPI {
     example: "(to-int u238) ;; Returns 238"
 };
 
+const BUFF_TO_INT_LE_API: SimpleFunctionAPI = SimpleFunctionAPI {
```

Comment:
Do we want to annotate the documentation somehow to indicate that this is only available in Clarity2?  Doesn't have to be in this PR, but if not, can you make a separate issue so we remember to do this?

Reply:
see https://github.com/blockstack/stacks-blockchain/issues/2716

I also added some text for now:
`    Note: This function is only available starting with Stacks 2.1.`